.github/CONTRIBUTING.md

```diff
@@ -0,0 +1,257 @@
+
+You can check whether your code complies with the coding standard using the `composer check-cs` command from the project root.
+
+# Considerations when writing sniffs
+
+## Unit Testing
+
+All PRs which affect sniffs, whether bug fixes to existing sniffs, or the addition of a new sniff, should be accompanied by unit tests.
+
+### Pre-requisites
+
+* PHP 5.4 or higher
+
+### Composer setup
+
+Use Composer to install all the necessary dependencies to write and run unit tests. Running
+
+```bash
+composer install
+```
+
+from the root of the cloned repository, will install `PHP_CodeSniffer`, `WordPress Coding Standards`, `PHPUnit`, `PHPCompatibility` and `security advisories`, which ensures that you won't install dependencies with known security vulnerabilities.
+
+### Other setups
+
+If you have PHP_CodeSniffer (PHPCS) and/or WordPress Coding Standards (WordPressCS) already installed, for instance as git clones, because you either contribute to these libraries as well or you want to develop for or test WPThemeReview with bleeding edge versions of either PHP_CodeSniffer or WordPress Coding Standards, you need to take some additional steps to make it all work.
+
+First, make sure you also have PHPCompatibility installed and make sure the `installed_paths` for PHP_CodeSniffer is setup correctly.
+
+You can see how this can be done by reading the official PHPCS [documentation](https://github.com/squizlabs/PHP_CodeSniffer/wiki/Configuration-Options#setting-the-installed-standard-paths).
+
+#### Method 1
+
+Set up global environment variables to point to PHP_CodeSniffer and WordPress Coding Standards:
+
+```bash
+PHPCS_DIR: I:/path/to/PHP_CodeSniffer/
+WPCS_DIR: I:/path/to/WordPressCS/
+```
+
+If you do that, both the sniffs as well as the unit tests should be able to work correctly.
+
+#### Method 2
+
+If you do not want to set up environment variables on your system
+
+* Add a file called `.pathtowpcs` to the project root. The only content in that file should be the full absolute path to your WordPressCS install.
+* Copy the `phpunit.xml.dist` file, rename the copied file `phpunit.xml` and make sure it's in the project root.
+Now add the following to that file, adjusting the paths to reflect those on your system:
+
+```xml
+  <php>
+    <env name="PHPCS_DIR" value="path/to/PHP_CodeSniffer/"/>
+    <env name="WPCS_DIR" value="path/to/WordPressCS/"/>
+  </php>
+```
+
+(and don't remove the existing line within the `<php>` block.)
+
+Once you've done that, both running the sniffs as well as the unit tests should work correctly.
+
+## Writing and running unit tests
+
+The most important thing when writing sniffs intended for the theme review, is to have ample example code. This makes writing sniffs a lot easier, because you can test against the given examples.
```

Comment:
```suggestion
The most important thing when writing sniffs intended for the theme review standard, is to have ample example code. This makes writing sniffs a lot easier, because you can test against the given examples.
```